lean4-infoview/src/infoview/info.tsx

```diff
@@ -166,10 +166,10 @@ export function InfoDisplay(props0: InfoDisplayProps) {
             </div>
             {nothingToShow && (
                 isPaused ?
-                    <span>Updating is paused.
+                    <span>Updating is paused.{' '}
```

Comment:
Can you add a comment here explaining why we need to do the `{' '}` trick?  Something like whitespace is stripped by the typescript compiler here which causes the anchor tags to show up with no space around them, adding a typescript expression containing a string literal makes it something that is not stripped automatically.

Reply:
Sure :)